Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM frolvlad/alpine-python3
```

Comment:
I did try to get an `apk add python3` version working, but it would still end up running `setup.py` with Python 2 and crashing.